client/hosting/overview/components/test/migration-overview.tsx

```diff
@@ -1,115 +0,0 @@
-/**
```

Comment:
The tests are now on `client/hosting/overview/components/migration-overview/test/index.tsx`